site/content/docs/concepts/services.en.md

```diff
@@ -25,13 +25,22 @@ When you're setting up a service, Copilot will ask you about what kind of servic
 If you want your service to serve internet traffic then you have two options:
 
 * "Request-Driven Web Service" will provision an AWS App Runner Service to run your service. 
-* "Load Balanced Web Service" will provision an Application Load Balancer, security groups, an ECS service on Fargate to run your service.
+* "Load Balanced Web Service" will provision an Application Load Balancer, a Network Load Balancer or both, along with 
+  security groups, an ECS service on Fargate to run your service.
 
 #### Request-Driven Web Service
 An AWS App Runner service that autoscales your instances based on incoming traffic and scales down to a baseline instance when there's no traffic. This option is more cost effective for HTTP services with sudden bursts in request volumes or low request volumes.
 
 #### Load Balanced Web Service
-An ECS Service running tasks on Fargate with an Application Load Balancer as ingress. This option is suitable for HTTP services with steady request volumes that need to access resources in a VPC or require advanced configuration.
+An ECS Service running tasks on Fargate with an Application Load Balancer, a Network Load Balancer or both, as ingress. 
+This option is suitable for HTTP or TCP services with steady request volumes that need to access resources in a VPC or 
+require advanced configuration. 
```

Comment:
This doesn't get renderred as a real newline in the doc. It's rendered as a newline only if there are two newlines here. I did this just because it's easier to see in the editor.